crates/holochain/src/conductor/handle.rs

```diff
@@ -412,14 +416,49 @@ impl<DS: DnaStore + 'static> ConductorHandleT for ConductorHandleImpl<DS> {
                     .map_err(holochain_p2p::HolochainP2pError::other);
                 respond.respond(Ok(async move { res }.boxed().into()));
             }
+            PutMetricDatum {
+                span_context,
+                respond,
+                dna_hash,
+                to_agent,
+                metric,
+            } => {
+                let env = { self.conductor.read().await.p2p_env(space) };
+                let agent = todo!("can we use to_agent?");
+                let res = put_metric_datum(env, agent, metric, SystemTime::now())
+                    .map_err(holochain_p2p::HolochainP2pError::other);
```

Comment:
I think I need direct control of the timestamp from the kitsune side, so if I start a sync at X, and the query to the remote side takes 2.3 seconds to succeed, I still mark the sync success time as X.